src/core/expression/qgsexpressionfunction.cpp

```diff
@@ -5961,8 +5962,8 @@ const QList<QgsExpressionFunction *> &QgsExpression::Functions()
         << new QgsStaticExpressionFunction( QStringLiteral( "array" ), -1, fcnArray, QStringLiteral( "Arrays" ), QString(), false, QSet<QString>(), false, QStringList(), true )
         << new QgsStaticExpressionFunction( QStringLiteral( "array_sort" ), QgsExpressionFunction::ParameterList() << QgsExpressionFunction::Parameter( QStringLiteral( "array" ) ) << QgsExpressionFunction::Parameter( QStringLiteral( "ascending" ), true, true ), fcnArraySort, QStringLiteral( "Arrays" ) )
         << new QgsStaticExpressionFunction( QStringLiteral( "array_length" ), QgsExpressionFunction::ParameterList() << QgsExpressionFunction::Parameter( QStringLiteral( "array" ) ), fcnArrayLength, QStringLiteral( "Arrays" ) )
-        << new QgsStaticExpressionFunction( QStringLiteral( "array_contains" ), QgsExpressionFunction::ParameterList() << QgsExpressionFunction::Parameter( QStringLiteral( "array" ) ) << QgsExpressionFunction::Parameter( QStringLiteral( "value" ) ), fcnArrayContains, QStringLiteral( "Arrays" ) )
-        << new QgsStaticExpressionFunction( QStringLiteral( "array_all" ), QgsExpressionFunction::ParameterList() << QgsExpressionFunction::Parameter( QStringLiteral( "array_a" ) ) << QgsExpressionFunction::Parameter( QStringLiteral( "array_b" ) ), fcnArrayAll, QStringLiteral( "Arrays" ) )
+        << new QgsStaticExpressionFunction( QStringLiteral( "array_contains" ), QgsExpressionFunction::ParameterList() << QgsExpressionFunction::Parameter( QStringLiteral( "array" ) ) << QgsExpressionFunction::Parameter( QStringLiteral( "value" ) ), fcnArrayContains, QStringLiteral( "Arrays" ), QString(), false, QSet<QString>(), false, QStringList(), false, QStringList() << QStringLiteral( "exists" ) )
```

Comment:
Can the tags be made translatable?

Reply:
I don't know whether it's the same context but when translating, Processing tags are shown as a comma separated list, eg `join,connect,attributes,values,fields,tables` that one could translate into `join,connect,attributes,values,fields,tables,joindre,connecter,valeurs,champs` convenient when you remember a tag in English while on a French QGIS. This is because the string is identifiable as a tag list and not as separate words one would need to translate. Is that kept or possible to keep?

Reply:
I plan to change it a little bit (see my last commit) and the tags will be held in the `Help` object, so the function is created regarding the files defined in `resources/function_help/json/`

When there is defined: 
```
"tags": [ "split", "convert", "seperate" ]
```
A function is created in `qgsexpression_texts.cpp` with a list like:
```
            QStringList()
              << QStringLiteral( "split" ) << tr( "split" )
              << QStringLiteral( "convert" ) << tr( "convert" )
              << QStringLiteral( "seperate" ) << tr( "seperate" )
```
So you translate it like usual and the English tag is kept.